src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Color[] colors = Color.values();
+    private static final Random random = new Random();
+
+    public String getRandomColor() {
```

Comment:
Don't use static methods in your solution. While 'getRandomColor' is not static, the 'random' object is declared as static. It's better to create an instance of a class which method you want to call.